client/web/src/components/views/login-view.tsx

```diff
@@ -14,9 +16,16 @@ export default function LoginView({
   data: NodeData
   refreshData: () => void
 }) {
+  const [controlURL, setControlURL] = useState<string>("")
+  const [authKey, setAuthKey] = useState<string>("")
+
   const login = useCallback(
     (opt: TailscaleUpOptions) => {
-      tailscaleUp(opt).then(refreshData)
+      tailscaleUp(opt).then(() => {
+        // Particularly when doing non-interactive auth with a key,
+        // give a little extra time to complete registration.
+        setTimeout(refreshData, 1000)
```

Comment:
Maybe only do it conditionally in the key case?

Reply:
yeah, I can do that.  There's also no harm really since in the non-key case we're typically opening a new window to login to control, so this is all happening in a background window.

Reply:
actually, I don't need this at all.  I hadn't noticed that the `/api/up` endpoint wasn't returning because it was waiting on an authURL.  I now have it waiting and returning once tailscale is running as well (which covers the authkey) use case, so by the time we reach this point when using an auth key, everything should be complete.  Tested, and seems to work well.